ros_ws/src/simulation/racer_sensors/src/racer_odometry.cpp

```diff
@@ -0,0 +1,48 @@
+#include <racer_odometry.h>
+
+RacerOdometry::RacerOdometry()
+{
+    m_gazebo_subscriber =
+        m_node_handle.subscribe<gazebo_msgs::LinkStates>(TOPIC_GAZEBO_ODOM, 1, &RacerOdometry::robotPoseUpdate, this);
+
+    m_odom_publisher = m_node_handle.advertise<nav_msgs::Odometry>(TOPIC_ODOM, 1);
+
+    m_last_received_pose = geometry_msgs::Pose();
+    m_last_received_twist = geometry_msgs::Twist();
+    m_last_received_stamp = ros::Time();
+
+    m_timer = m_node_handle.createTimer(ros::Duration(0.05), boost::bind(&RacerOdometry::timerCallback, this, _1));
+}
+
+void RacerOdometry::robotPoseUpdate(const gazebo_msgs::LinkStates::ConstPtr& links)
+{
+    for (uint i = 0; i < links->name.size(); i++)
```

Comment:
Where does the type `uint` comes from? How is it defined?

Reply:
Changed `uint` to `unsigned int`. They are synonymous but I get that it is bad form.